recipes/geographiclib/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class GeographiclibConan(ConanFile):
+    name = "geographiclib"
+    description = "Convert geographic units and solve geodesic problems"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://geographiclib.sourceforge.io"
+    license = "MIT"
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    exports_sources = ["CMakeLists.txt"]
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+    _cmake = None
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            self.options.remove("fPIC")
```

Comment:
```suggestion
            del self.options.fPIC
```

Both have the same behavior, but `del` is more pythonic